visualization/include/pcl/visualization/vtk/vtkRenderWindowInteractorFix.mm

```diff
@@ -41,184 +41,7 @@
 #include <vtkCocoaRenderWindowInteractor.h>
 #include <vtkObjectFactory.h>
 
-#if ((VTK_MAJOR_VERSION < 6) || ((VTK_MAJOR_VERSION == 6) && (VTK_MINOR_VERSION < 2)))
```

Comment:
This check is for anything less than 6.2 and we set VTK to have a minimum version of 6.0. Either we bump the minimum version to 6.2 or we cannot erase it. @PointCloudLibrary/maintainers .

Just from seeing this code I would say we bump it to 6.2.

Reply:
Ubuntu 16.04 comes with 6.3, so I think it's okay.